### PR TITLE
Created new Genre page

### PR DESCRIPTION
**Pull request by roundband5**
New Genre page with top 20 songs playlist

ezgif-6-270c9efa5e0d

### CODE REVIEW

File: src/modules/Genre/Genre.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+  .row {
+    display: grid;
+    align-items: center;
+    grid-auto-flow: column;
+  }
+
+  .image-col{
```
Comment (leonzh2k) [inline src/modules/Genre/Genre.css:7]: Fix spacing (one space between selector and bracket)

File: src/modules/Genre/Genre.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  }
+  
+  .col {
+    align-items: center;
+    width: 250px;
+  }
+  .genre-description{
+    text-align: center;
+    align-items: center;
+    font-size:1.5em;
```
Comment (leonzh2k) [inline src/modules/Genre/Genre.css:30]: Fix spacing

File: src/modules/Genre/Genre.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  .genre-details {
+    gap: 5%;
+    margin-top: 2%;
+    margin-bottom: 2%;
+    margin-left: -17%;
+  }
+  
+  .genre-header {
+    font-weight: 600;
+    font-size:800%;
```
Comment (leonzh2k) [inline src/modules/Genre/Genre.css:43]: Spacing

**Review by leonzh2k — CHANGES_REQUESTED** [review]
Looks good except for inconsistent spacing.

**Review by lchua2314 — APPROVED** [review]
Make leonzh2k changes then it's good.

**Comment by roundband5** [comment]
Fixed spacing issues in code
improved website readability

**Review by leonzh2k — APPROVED** [review]
Looks good, merging

**Pull request merged**